### DISCUSSION TITLE
"achievements" generates error

### DISCUSSION THREAD

**Discussion by dkhokhlov** [lowlighter/metrics#1505] [Q&A] [answered]
"achievements" generates error
It got broken a month ago.  You can see error on https://github.com/lowlighter profile page too.

**Comment by Vault108** [comment, marked as answer]
[#1479](https://github.com/lowlighter/metrics/issues/1479)
Switching branches to master will fix it

**Comment by lowlighter** [comment]
It should now work on @latest 🙂
